agent/api/task/task.go

```diff
@@ -1535,6 +1540,11 @@ func (task *Task) dockerHostConfig(container *apicontainer.Container, dockerCont
 				hostConfig.ExtraHosts = append(hostConfig.ExtraHosts, hosts...)
 			}
 
+			if task.shouldEnableIPv6() {
+				// By default, the disable ipv6 setting is turned on, so need to turn it off to enable it.
+				enableIPv6SysctlSetting(hostConfig)
```

Comment:
Can you please check with @aaithal to make sure this sysctl won't execute on Fargate? I don't know if this package is shared. My understanding is this is to undo the value set by Docker, and Fargate doesn't need it because it uses containerd.

Reply:
Ack. This shouldn't affect Fargate.